plugins/polio/admin.py

```diff
@@ -75,6 +78,42 @@ class CampaignGroupAdmin(admin.ModelAdmin):
     pass
 
 
+class MailTemplateAdmin(admin.ModelAdmin):
+    pass
+
+
+class BudgetStepLinkAdminInline(admin.TabularInline):
+    model = BudgetStepLink
+    extra = 0
+
+
+class BudgetStepFileAdminInline(admin.TabularInline):
+    model = BudgetStepFile
+    extra = 0
+
+
+class BudgetStepAdmin(admin.ModelAdmin):
+    inlines = [
+        BudgetStepFileAdminInline,
+        BudgetStepLinkAdminInline,
+    ]
+    list_display = ["campaign", "transition_key", "created_by", "created_at", "deleted_at"]
+
+
+class WorkflowAdmin(admin.ModelAdmin):
+    readonly_fields = [
+        "pretty_json",
+    ]
+
+    def pretty_json(self, obj: WorkflowModel):
+        try:
+            d = json.dumps(obj.definition, indent=2)
+            html = f"<pre>{d}</pre>"
+            return mark_safe(html)
+        except Exception as e:
+            print(e)
```

Comment:
Maybe we can catch a less broad exception, and use the logging module rather than print().